Userland/Services/LoginServer/LoginWindow.gml

```diff
@@ -25,6 +25,12 @@
         @GUI::Widget {
             layout: @GUI::HorizontalBoxLayout {}
 
+            @GUI::Label {
+                name: "fail_message"
+                foreground_color: "red"
```

Comment:
The system is themeable, so let's not use hard-coded colors. Instead, you can set `foreground_role` to one of the `Gfx::ColorRole` enum values.
However, in this case I'm not sure a custom color is necessary in the first place(?)

Reply:
Yep, the black message certainly looks a bit better :)